src/components/comment-media/comment-media.module.css

```
@@ -55,14 +55,31 @@
 }
 
 .thumbnailSmall span,
-.thumbnailSmall a {
+.thumbnailSmall a,
+.thumbnailSmall .noThumbnailButton {
   width: 100%;
   display: inline-block;
   word-wrap: break-word;
```

Stylelint (17.12.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)